Formula/vips.rb

```diff
@@ -25,7 +25,6 @@ class Vips < Formula
   depends_on "little-cms2"
   depends_on "orc"
   depends_on "pango"
-  depends_on "pygobject3"
```

Comment:
why is this being removed?

Reply:
Sorry, I hadn't noticed that change. I don't think it should be there, but that's clearly a separate issue. I'll revise again.